t/lib/OpenQA/Test/Utils.pm

```diff
@@ -467,13 +465,14 @@ sub unstable_worker {
             });
         setup_worker($worker, $host);
         $worker->init();
+        Devel::Cover::report() if Devel::Cover->can('report');
         if ($ticks < 0) {
+            # This call will effectively never return
             Mojo::IOLoop->singleton->start;
         }
         else {
             Mojo::IOLoop->singleton->one_tick for (0 .. $ticks);
```

Comment:
as you moved the report line above this one would also never be collected in coverage